DataFormats/Phase2TrackerCluster/interface/Phase2TrackerCluster1D.h

```diff
@@ -0,0 +1,43 @@
+#ifndef DATAFORMATS_PHASE2TRACKERCLUSTER_PHASE2TRACKERCLUSTER1D_H 
+#define DATAFORMATS_PHASE2TRACKERCLUSTER_PHASE2TRACKERCLUSTER1D_H 
+
+#include <stdint.h>
```

Comment:
@Dr15Jones 
should we be moving to cstdint (C++ header) instead for the fixed width types?